test/api_integration/apis/saved_objects/create.ts

```diff
@@ -51,6 +51,7 @@ export default function ({ getService }: FtrProviderContext) {
             type: 'visualization',
             coreMigrationVersion: '8.8.0',
             typeMigrationVersion: resp.body.typeMigrationVersion,
+            managed: resp.body.managed,
```

Comment:
Same here too.

Reply:
same question as in https://github.com/elastic/kibana/pull/154515#discussion_r1165098941